src/CompletionItem.php

```diff
@@ -176,13 +193,14 @@ class CompletionItem
      * @param string|null $filterText
      * @param string|null $insertText
      * @param InsertTextFormat::*|null $insertTextFormat
-     * @param TextEdit|null $textEdit
+     * @param InsertTextMode::*|null $insertTextMode
+     * @param TextEdit|InsertReplaceEdit|null $textEdit
      * @param array<TextEdit>|null $additionalTextEdits
      * @param array<string>|null $commitCharacters
      * @param Command|null $command
      * @param mixed|null $data
      */
-    public function __construct(string $label, $kind = null, ?array $tags = null, ?string $detail = null, $documentation = null, ?bool $deprecated = null, ?bool $preselect = null, ?string $sortText = null, ?string $filterText = null, ?string $insertText = null, $insertTextFormat = null, ?TextEdit $textEdit = null, ?array $additionalTextEdits = null, ?array $commitCharacters = null, ?Command $command = null, $data = null)
+    public function __construct(string $label, $kind = null, ?array $tags = null, ?string $detail = null, $documentation = null, ?bool $deprecated = null, ?bool $preselect = null, ?string $sortText = null, ?string $filterText = null, ?string $insertText = null, $insertTextFormat = null, $insertTextMode = null, $textEdit = null, ?array $additionalTextEdits = null, ?array $commitCharacters = null, ?Command $command = null, $data = null)
```

Comment:
Possibly problematic B/C break (if any of the code is using the constructor directly)